lib/Conversion/TorchToLinalg/Linear.cpp

```diff
@@ -415,6 +415,42 @@ class ConvertAtenMatmulOp : public OpConversionPattern<AtenMatmulOp> {
 };
 } // namespace
 
+namespace {
+class ConvertAtenMvOp : public OpConversionPattern<AtenMvOp> {
```

Comment:
I think it would make more sense to add a decomposition of this op into `AtenMatmulOp`, since that op performs this same handling:

https://github.com/llvm/torch-mlir/blob/9dd5ae8239e2d45fd8959b8b014bf0c76be16d31/lib/Conversion/TorchToLinalg/Linear.cpp#L217

The decomposition would happen in the `DecomposeComplexOps.cpp` file